exercises/transpose/transpose_test.py

```diff
@@ -0,0 +1,275 @@
+import unittest
+from transpose import transpose
+
+
+class TransposeTests(unittest.TestCase):
+    def test_empty_string(self):
+        input_line = ""
+        expected = ""
+        self.assertEqual(
+            expected,
+            transpose(input_line)
+        )
+
+    def test_two_characters_in_a_row(self):
+        input_line = ["A1"]
+        expected = [
```

Comment:
I think it would be more readable if this is in one line `expected = ["A", "1"]`
... or when the input is that small maybe even without the variables.
```python
self.assertEqual("\n".join(["A", "1"]), transpose("A1"))
```